action.yml

```diff
@@ -0,0 +1,23 @@
+name: Set up
```

Comment:
I've been trying to get this nested under `.github`, but haven't been able to get it working properly. The convention seems to be to have composite actions like these in the root level of a repo, but I can't find any explicit requirements. 

Really, the cleanest and easiest would be to put this in a separate repo entirely. It would also be more extendable if we end up needing more custom composite actions. Thoughts?

Reply:
Agreed! Also gets rid of the very opaque setup of referring using `./`